Products/ZPyODBCDA/pyodbc_db.py

```diff
@@ -276,7 +276,7 @@ def _abort(self):
         try:
             self._conx.rollback()
         except DB_Error:
-            raise
+            self._conx.close()
```

Comment:
Shouldn't we still raise after closing the connection?

Reply:
I guess not really, since we're trying to re-establish the connection and not throw an error